### ISSUE TITLE
Popups are blurred when the script is working

### ISSUE THREAD

**Issue by gizellos** [revilheart/ESGST#347]
Popups are blurred when the script is working. It is a new issue. It's old a day or two.

![blurry pop out](https://user-images.githubusercontent.com/2397851/29317672-52db7f7e-81cd-11e7-82e0-1f3d7502de54.gif)

**Comment by revilheart** [comment]
Should be fixed.

**Comment by gizellos** [comment]
Unfortunately, it didn't help it.

**Comment by gizellos** [comment]
And it broke the pop out from the gif and I can see there a horizontal scrollbar after it is finished.

![2017-08-15_161748](https://user-images.githubusercontent.com/2397851/29319751-4ef6c58c-81d5-11e7-97e0-0b8433847be3.png)

**Comment by revilheart** [comment]
I don't see anything broke, what do you mean by that?

**Comment by gizellos** [comment]
The pop out is not wide enough to show all the data without the horizontal scrollbar. It was OK before the last version. I have resolution of monitor 1920/1080. There is plenty of space for the whole table.

**Comment by gizellos** [comment]
It seems the blurred text is really fixed in most of pop outs but it didn't help in the  Sent/Won giveaways data.

**Comment by gizellos** [comment]
The blurred content is still issue in Highlighted Discussions, Sent/Won giveaways data and Comment History and Decrypted Giveaways. I just tried it in v6.Beta.27.1.

Could you please reopen this issue?

**Comment by revilheart** [comment]
I'm personally not getting any blur anywhere.

**Comment by gizellos** [comment]
If you were a user and not a developer himself, I would ask you if you have the newest version, because else I don't understand why you don't see it. :D :(

I just tested it on another computer to be sure it is not because of my computer or the specific settings I use.

I turned on the other computer, installed Tampermonkey, installed ESGST, logged to Steamgifts, turned on only feature "User Giveaways Data" and tried to get Sent data of some random users on SG. There was always the blurring. 

Maybe it occurs only on systems with Nvidia and Intel GPUs? Or the blur is only on OS Windows?
